v03_pipeline/lib/tasks/delete_family_tables.py

```diff
@@ -1,10 +1,12 @@
 import luigi
 
+from v03_pipeline.lib.model import SampleType
 from v03_pipeline.lib.tasks.base.base_hail_table import BaseHailTableTask
 from v03_pipeline.lib.tasks.delete_family_table import DeleteFamilyTableTask
 
 
 class DeleteFamilyTablesTask(BaseHailTableTask):
+    sample_type = luigi.EnumParameter(enum=SampleType)
```

Comment:
@bpblanken fyi, if we want to delete all family or project tables for a family or project, we have to run the delete dag once with each sample type